### PR TITLE
Query Editor (NoSQL + Mongo)

### PR DESCRIPTION
**Pull request by bk201-**
Tasks:
* [x] Fix Packaging
* [x] Fix Debugging with VS Code
* [x] Apply vCore feature flag for vCore commands

### CODE REVIEW

File: src/panels/ResultPanel.ts
Comment (languy) [inline src/panels/ResultPanel.ts:1]: Do we need this empty file?

File: src/panels/ResultTab.ts
Comment (languy) [inline src/panels/ResultTab.ts:1]: Do we need this empty file?

File: src/webviews/QueryEditor/ResultPanel/DataViewPanelJSON.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    const getVscodeTheme = (themeKind: string) => {
+        return themeKind === 'vscode-light'
+            ? 'vs'
+            : themeKind === 'vscode-dark'
+              ? 'vs-dark'
+              : themeKind === 'vscode-high-contrast'
+                ? 'hc-black'
+                : themeKind === 'vscode-high-contrast-light'
+                  ? 'hc-light'
+                  : 'light';
```
Comment (languy) [inline src/webviews/QueryEditor/ResultPanel/DataViewPanelJSON.tsx:28]: Is it the same as in `QueryMonaco.tsx`? If so, move this in some util module?

File: src/webviews/QueryEditor/ResultPanel/InsightsTab.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+                    </tr>
+                    <tr>
+                        <td>Visibility</td>
+                        <td>0.5 miles, 1800 ft runway visual range</td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+    );
+};
```
Comment (languy) [inline src/webviews/QueryEditor/ResultPanel/InsightsTab.tsx:26]: This does not seem to be used. Remove?

File: src/webviews/theme/DynamicThemeProvider.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    return themeKind === 'vscode-light'
+        ? teamsLightTheme
+        : themeKind === 'vscode-dark'
+          ? teamsDarkTheme
+          : themeKind === 'vscode-high-contrast'
+            ? teamsHighContrastTheme
+            : themeKind === 'vscode-high-contrast-light'
+              ? teamsLightTheme // TODO: find a better theme for this
+              : undefined;
+};
```
Comment (languy) [inline src/webviews/theme/DynamicThemeProvider.tsx:59]: Plug this in `QueryMonaco.tsx` and `DataViewPanelJSON.tsx`?

**Review by languy — APPROVED** [review]

**Review by sevoku — APPROVED** [review]

**Pull request merged**